packages/api-docs-builder-core/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@mui-internal/api-docs-builder-core",
```

Comment:
I was browsing the source and saw this. This is very strange. It looks like this should be in the /scripts/ folder.

Reply:
> npm packages introduce problems as soon as created

What problems do you have in mind?